arangod/RocksDBEngine/RocksDBKeyBounds.cpp

```diff
@@ -415,6 +423,9 @@ RocksDBKeyBounds::RocksDBKeyBounds(RocksDBEntryType type, uint64_t first,
     : _type(type) {
   switch (_type) {
     case RocksDBEntryType::ZkdIndexValue:
+    case RocksDBEntryType::UniqueZkdIndexValue:
+    case RocksDBEntryType::ZkdVPackIndexValue:
+    case RocksDBEntryType::UniqueZkdVPackIndexValue:
     case RocksDBEntryType::VPackIndexValue:
     case RocksDBEntryType::UniqueVPackIndexValue: {
       uint8_t const maxSlice[] = {0x02, 0x03, 0x1f};
```

Comment:
Is it actually ok to use a VelocyPack `max` value for the upper bound (after the index id) in case of the traditional ZkdIndex?
I was under the impression that the traditional Zkd index does not store VelocyPack data in the keys, and not use a VelocyPack-aware comparator. So using a VelocyPack `max` value as part of the upper will likely not work as expected.